src/model/pad_options/options.go

```diff
@@ -125,16 +156,37 @@ func delete_file(w http.ResponseWriter ,r *http.Request, _ httprouter.Params){
 
 	if val,ok :=PadMap[t.ID]; ok {
 			fmt.Println("Delete", val.Name)
-		delete(PadMap,t.ID)
-		err := os.Remove(PadMap[t.ID].ID)
+		err := os.Remove("./SavedFiles/"+PadMap[t.ID].ID+".txt")
 		if err != nil {
 			log.Fatal(err)
 		}
+		delete(PadMap,t.ID)
+		
 
 	}else{
 		fmt.Println("File %s not found",t.ID)
 	}
 }
 
+func (p Pad) EmptyDocument(w http.ResponseWriter ,r *http.Request, _ httprouter.Params){
+	decoder := json.NewDecoder(r.Body)
+	var t Pad
+	err := decoder.Decode(&t)
+	if err != nil {
+		panic(err)
+	}
+
+	if val,ok :=PadMap[t.ID]; ok {
+		fmt.Println("Empty Document : ", val.Name)
+		err := os.Truncate("./SavedFiles/"+PadMap[t.ID].ID+".txt", 0)
```

Comment:
nice but better error handling server must never close but return to client what went wrong, appropriate header: [https://en.wikipedia.org/wiki/List_of_HTTP_status_codes](url) and maybe a msg in json